pkgs/applications/networking/instant-messengers/gtkcord/default.nix

```
@@ -0,0 +1,46 @@
+{ makeDesktopItem, lib, buildGoModule, fetchFromGitHub, pkg-config, glib, go, gtk3, libhandy }:
+
+buildGoModule rec {
+  pname = "gtkcord";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "diamondburned";
+    repo = "gtkcord3";
+    rev = "v${version}";
+    sha256 = "0n9600s50zpx8kv89icrlxarm3633gnizg1sq1zpask8bbv2xnfd";
+  };
+
+  nativeBuildInputs = [ pkg-config go ];
+  
+  buildInputs = [ glib gtk3 libhandy ];
+
+  desktopFile = makeDesktopItem {
+	  name = pname;
+    desktopName = "gtkcord";
+	  exec = "gtkcord3";
+	  icon = "gtkcord3";
+	  categories = "GTK;GNOME;Chat;";
+  };
+
+	preFixup = ''
+    # Create directory
+    install -d $out/share/applications/
+	  # Install the desktop file
+	  install -D "${desktopFile}"/share/applications/* $out/share/applications/
+	  # Install the icon
+	  install -D "$src/logo.png" $out/share/icons/hicolor/256x256/apps/gtkcord3.png
+  '';
+
+  vendorSha256 = "0kzq8x0qcjr443ydj05s2f8ybszc2jzqa2annadp3hry8x2va2in";
+
+  subPackages = [ "." ];
+
+  meta = with lib; {
+    description = "Lightweight Discord client which uses GTK3 for the user interface";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ skykanin ];
+    homepage = "https://github.com/diamondburned/gtkcord3";
+    platforms = [ "x86_64-darwin" "x86_64-linux" ];
```

It is better to use attrs in `lib.platforms` so that we get an error when it does not exist. 
```suggestion
    platforms = [ platforms.x86_64-darwin platforms.x86_64-linux ];
```

---

There are no such attributes in `platforms` instead I opted for `platforms.darwin ++ platforms.linux`

---

Right, sorry.